step-capstone/src/components/Firebase/firebase.js

```diff
@@ -55,6 +55,51 @@ class Firebase {
     });
   }
 
+  getData(reference){
```

Comment:
This collection of functions is a little confusing because there's no overriding organizational scheme. You're going to end up with a lot of these and consistency will make it easier to follow. What I might recommend is having a get, add, edit and delete for each type of object with the same naming pattern. For example, `getTravel`, `addTravel`, `editTravel`, `deleteTravel`. These could then call private helper methods as needed.

Alternatively, it might make sense to put common helpers here like `getCollection` and `getData`, then do the actual implementations for each object in that object's class.

Reply:
good idea, I will switch to using a common naming scheme.